app/views/admin/orders/_order_products_table.html.erb

```diff
@@ -0,0 +1,38 @@
+<table id="proucts-table" class="table table-sm">
+  <thead>
+    <tr>
+      <th scope="col" >Product</th>
+      <th scope="col" >Brand</th>
+      <th scope="col" >Category</th>
+      <th scope="col" >Price</th>
+    </tr>
+  </thead>
+  <tbody scope="col">
+    <% order.cart_items.includes([:product]).each do |item| %>
```

Comment:
Co myślisz o
```suggestion
    <% order.cart_items.includes([:product]).map(&:product).each do |product| %>
```
Myślałem, żeby zrobić `order.products` ale wtedy jeśli dodam kilka sztuk tego samego produktu, to wyświetli tylko pojedyńcze.

Reply:
Ale może to miałoby sens, gdyby w przyszłości dodać po prostu kolumnę `count`, czy coś w tym stylu.

Reply:
```order.cart_items.includes([:product])``` to imo powinno przychodzic wyzej. Nie jestem przekonany czy w widoku powinny byc wywolywane takie metody AR. Masz jakis powod dla ktorego w tym miejscu musi byc ten include?

Reply:
co myslisz o stworzeniu zmiennej product (w L#12) ?
```
<% product = item.product %>
```

Reply:
dobry pomysl, ale zmienilem ta zapytanie do `order.products` i teraz przekazuję do tego partiala tylko zmienna `products`, bo ta tabela nie porzebuje więcej danych.